diff --git a/node_modules/@nestjs-modules/mailer/dist/interfaces/mailer-options.interface.d.ts b/node_modules/@nestjs-modules/mailer/dist/interfaces/mailer-options.interface.d.ts
index e0847c8..5f9f1cc 100644
--- a/node_modules/@nestjs-modules/mailer/dist/interfaces/mailer-options.interface.d.ts
+++ b/node_modules/@nestjs-modules/mailer/dist/interfaces/mailer-options.interface.d.ts
@@ -1,10 +1,10 @@
-import { Transport, TransportOptions } from 'nodemailer';
-import * as JSONTransport from 'nodemailer/lib/json-transport';
-import * as SendmailTransport from 'nodemailer/lib/sendmail-transport';
-import * as SESTransport from 'nodemailer/lib/ses-transport';
-import * as SMTPPool from 'nodemailer/lib/smtp-pool';
-import * as SMTPTransport from 'nodemailer/lib/smtp-transport';
-import * as StreamTransport from 'nodemailer/lib/stream-transport';
+import type { MailDefaults, Transport, TransportOptions } from 'nodemailer';
+import type JSONTransport from 'nodemailer/lib/json-transport';
+import type SendmailTransport from 'nodemailer/lib/sendmail-transport';
+import type SESTransport from 'nodemailer/lib/ses-transport';
+import type SMTPPool from 'nodemailer/lib/smtp-pool';
+import type SMTPTransport from 'nodemailer/lib/smtp-transport';
+import type StreamTransport from 'nodemailer/lib/stream-transport';
 import { I18nOptions } from './i18n-options.interface';
 import { TemplateAdapter } from './template-adapter.interface';
 import { TemplateResolver } from './template-resolver.interface';
@@ -19,7 +19,7 @@ export interface RateLimitOptions {
     period?: number;
 }
 export interface MailerOptions {
-    defaults?: Options;
+    defaults?: MailDefaults;
     transport?: TransportType;
     transports?: {
         [name: string]: TransportType;
